src/permissions/PermissionController.test.ts

```diff
@@ -4408,7 +4408,7 @@ describe('PermissionController', () => {
 
       expect(
         Object.keys(
-          messenger.call('PermissionController:getPermissions', 'foo'),
+          messenger.call('PermissionController:getPermissions', 'foo') ?? {},
```

Comment:
Can't we just assert that its not null with `!` for the test?

Reply:
Not allowed due to eslint rules

Reply:
Perhaps we should just disable that rule for this specific instance. I don't think introducing logic that'll never be used is the solution

Reply:
Some of the ESLint rules, like this one, are meant to be broken in exceptional cases. The non-null assertion operator is blocked because it's commonly used to ignore legitimate type errors where the null case is being ignored, and where ignoring that possibility could result in an error.

There are plenty of cases where the non-null assertion operator is safe to use, and adding an eslint-ignore comment to allow using it would be perfectly fine. I usually accompany it with a comment explaining why it's safe as well, if it's not self-evident.